docs/contents/components/forms/hue-slider/accessibility.ja.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: HueSlider
+description: "`HueSlider`は、ユーザーが色相を選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/color-picker"
+order: 4
+with_description: true
+tab: アクセシビリティ
+---
+
+`HueSlider`に`aria-label`を設定すると、スクリーンリーダーによって読み上げられます。
+
+```tsx live=false
+<HueSlider aria-label="Hue slider" />
+```
```

Comment:
It is not read by screen reader.

Reply:
What did you use to check this? 
I'd like to understand better.